src/System.Diagnostics.Process/tests/ProcessTests.netstandard1.7.cs

```diff
@@ -15,7 +15,6 @@ namespace System.Diagnostics.Tests
 {
     public partial class ProcessTests : ProcessTestBase
     {
-        [PlatformSpecific(TestPlatforms.Windows)]
```

Comment:
Might be worthwhile adding a new test that uses RemoteInvoke to spawn a new process that checks HandleCount, then opens a bunch of files, refreshes and rechecks HandleCount, and makes sure that the new count is larger than the old one.